nesta/core/orms/nih_orm.py

```diff
@@ -2,116 +2,122 @@
+    award_notice_date = Column(DATETIME)
+    budget_start = Column(DATETIME)
+    budget_end = Column(DATETIME)
+    cfda_code = Column(TEXT)
+    core_project_num = Column(VARCHAR(50), index=True)
+    ed_inst_type = Column(TEXT)
+    foa_number = Column(TEXT)
+    full_project_num = Column(VARCHAR(50), index=True)
+    funding_ics = Column(JSON)
+    funding_mechanism = Column(TEXT)
+    fy = Column(INTEGER, index=True)
+    ic_name = Column(VARCHAR(100), index=True)
+    org_city = Column(VARCHAR(50), index=True)
+    org_country = Column(VARCHAR(50), index=True)
+    org_dept = Column(VARCHAR(100), index=True)
     org_district = Column(INTEGER)
-    org_duns = Column(VARCHAR(75))
-    org_fips = Column(VARCHAR(2))
+    org_duns = Column(JSON)
+    org_fips = Column(VARCHAR(2), index=True)
     org_ipf_code = Column(INTEGER)
-    org_name = Column(VARCHAR(92))
-    org_state = Column(VARCHAR(2))
+    org_name = Column(VARCHAR(100), index=True)
+    org_state = Column(VARCHAR(2), index=True)
     org_zipcode = Column(VARCHAR(10))
     phr = Column(TEXT)
-    pi_ids = Column(VARCHAR(262))
-    pi_names = Column(VARCHAR(533))
-    program_officer_name = Column(VARCHAR(39))
-    project_start = Column(VARCHAR(10))
-    project_end = Column(VARCHAR(10))
-    project_terms = Column(TEXT)
-    project_title = Column(VARCHAR(200))
+    pi_ids = Column(JSON)
+    pi_names = Column(JSON)
+    program_officer_name = Column(TEXT)
+    project_start = Column(DATETIME)
+    project_end = Column(DATETIME)
+    project_terms = Column(JSON)
+    project_title = Column(TEXT)
     serial_number = Column(VARCHAR(6))
     study_section = Column(VARCHAR(4))
-    study_section_name = Column(VARCHAR(95))
+    study_section_name = Column(TEXT)
     suffix = Column(VARCHAR(6))
     support_year = Column(VARCHAR(2))
     direct_cost_amt = Column(INTEGER)
     indirect_cost_amt = Column(INTEGER)
     total_cost = Column(INTEGER)
     subproject_id = Column(INTEGER)
     total_cost_sub_project = Column(INTEGER)
-    nih_spending_cats = Column(VARCHAR(2232))
+    nih_spending_cats = Column(JSON)
 
 
 class Abstracts(Base):
     __tablename__ = 'nih_abstracts'
 
-    application_id = Column(INTEGER, primary_key=True)
+    application_id = Column(INTEGER, primary_key=True, autoincrement=False)
     abstract_text = Column(TEXT)
 
+
 class Publications(Base):
     __tablename__ = 'nih_publications'
 
-    pmid = Column(INTEGER, primary_key=True)
-    author_name = Column(VARCHAR(166))
+    pmid = Column(INTEGER, primary_key=True, autoincrement=False)
+    author_name = Column(TEXT)
     affiliation = Column(TEXT)
-    author_list = Column(TEXT)
-    country = Column(VARCHAR(25))
+    author_list = Column(JSON)
+    country = Column(VARCHAR(50), index=True)
     issn = Column(VARCHAR(9))
     journal_issue = Column(VARCHAR(75))
-    journal_title = Column(VARCHAR(282))
-    journal_title_abbr = Column(VARCHAR(108))
+    journal_title = Column(VARCHAR(400), index=True)
```

Comment:
Crazy, I know, but there is a journal title with 279 chars:

`The quarterly journal of nuclear medicine and molecular imaging : official publication of the Italian Association of Nuclear Medicine (AIMN) [and] the International Association of Radiopharmacology (IAR), [and] Section of the Society of Radiopharmaceutical Chemistry and Biology.`

I like to give a bit of leeway in case a bigger title somehow comes out in the future...